charts/rancher-monitoring/102.0.2+up40.1.2/charts/kube-state-metrics/templates/servicemonitor.yaml

```diff
@@ -34,7 +34,7 @@ spec:
     {{- end }}
       metricRelabelings:
       {{- if .Values.prometheus.monitor.metricRelabelings }}
-        {{- toYaml .Values.prometheus.monitor.metricRelabelings | nindent 8 }}
+        {{- toYaml .Values.prometheus.monitor.metricRelabelings | nindent 6 }}
```

Comment:
This patch should probably be changed to fix the underlying problem:
```
+      {{ if .Values.global.cattle.clusterId }}
+      - sourceLabels: [__address__]
+        targetLabel: cluster_id
+        replacement: {{ .Values.global.cattle.clusterId }}
+      {{- end }}
+      {{ if .Values.global.cattle.clusterName }}
+      - sourceLabels: [__address__]
+        targetLabel: cluster_name
+        replacement: {{ .Values.global.cattle.clusterName }}
     {{- end }}
```
by indenting the rancher specific relabellings (currently at 6 spaces to 8) to match the upstream operator. This will prevent inadvertently re-introducing the bug in future upstream rebases.

Reply:
Done @alexandreLamarre

Reply:
Approved! Thank you for the patch